test_utils/utils.go

```diff
@@ -70,7 +70,7 @@ func WipeK8sPods() {
 	}
 }
 
-func RunIPAMPlugin(netconf, command, args string) (types.Result, int) {
+func RunIPAMPlugin(netconf, command, args string) (types.Result, types.Error, int) {
```

Comment:
Hmm good point. I don't see where this could be used where `exitCode` isn't enough. I will remove it.